pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "great_expectations_cloud"
-version = "20240528.0.dev0"
+version = "20240528.1.dev0"
```

Comment:
should this be a full release @anthonyburdi ?

Reply:
Yes it should be a full release - you can run `invoke release` and then commit the change it makes.